test/ng/ngPropSpec.js

```diff
@@ -49,6 +49,32 @@ describe('ngProp*', function() {
     expect(element.prop('asdf')).toBe(true);
   }));
 
+  // https://github.com/angular/angular.js/issues/16797
+  it('should support falsy property values', inject(function($rootScope, $compile) {
+    var element = $compile('<span ng-prop-text="myText" />')($rootScope);
+    // Initialze to non-falsey
+    $rootScope.myText = 'abc';
+    $rootScope.$digest();
```

Comment:
As I like to say, you can make these shorter with: `$rootScope.$apply('text = ...')` :grin: